config.properties.template

```diff
@@ -77,6 +78,9 @@ sort_by_iv=false
 # Always throw curveballs
 always_curve=false
 
+# Never use berries
+never_use_berries=true
```

Comment:
That is already the default behavior for days.

It will first try ever higher balls. Then add curve. Then add a berry.